### DISCUSSION TITLE
Issue with owners

### DISCUSSION THREAD

**Discussion by graschbaten** [bucanero/apollo-ps3#23] [Q&A] [answered]
Hi, 
does it work with the owners.xml?
I cant select any user in options and every created savegame has a new user so i can't use this tool

**Comment by bucanero** [comment]
what are you trying to do? it's not clear and hard to guess

in a normal scenario, `owners.xml` is not needed at all. You just install Apollo, run the app, and it detects all the IDs (account id, PSID, IDPS, user id). Then you patch and resign whatever save you have, and apollo will use the auto-detected ids. You can see the auto-detected values in the "About" menu.

unless you want to override values with hard-coded ones, there's no need for the owners file.

can you explain what do you want to do, and how are you trying to do it?

**Comment by graschbaten** [comment]
Hello Damián,

i want implant Cheats in the Savegame.
I start the app, choose my Savegame, choose the cheat and do apply and resign

In the the About section everything looks normal



This is the normal/regular Savegame




and this after i apply Changes & Resign

**Comment by bucanero** [comment]
are you trying to patch a save on HDD or USB?
Apollo gives any error when you click "apply & resign"?

not sure if you tried to attach a file or something, but nothing shows up on Github.

btw, have you tried with an older version of Apollo, for example v1.2.0. Do you have the same issue?
https://github.com/bucanero/apollo-ps3/releases/tag/v1.2.0

**Comment by graschbaten** [comment]
I got test it on USB and HDD with the same results also with Version 1.2.0 with same results.
I attach nothing. I have a clean Install with rebut 4.86.1 lite and i only install the latest Webman, Apollo, Artemis, PSN Patch(i never use this at this installation).
On this Console are no obscure Tweaks or hacks everything relativ clean and i never touch the idps. I only save the act.dat and idps.hex and nothing else
And no Apollo don’t give any Error. I mean this is right but i can’t understand why he changed the id!?

All my versions are from Github.

so can i test it if the users.xml is working for me? This isn’t the cleanest way but only for a test
Am 18.08.2020 um 17:31 schrieb Damián Parrino <notifications@github.com <mailto:notifications@github.com>>:

**Comment by bucanero** [comment, marked as answer]
I don't know what "ID" is being changed?... a savegame uses: account ID, user ID, and PSID. It's not using the detected values? Have you tried erasing the Apollo settings save?

if you have the `owners.xml` file, you need to place the file in:
`/dev_hdd0/game/NP0APOLLO/USRDIR/owners.xml`

next time you load Apollo, you can select the hardcoded values from the Settings menu.

the xml file should look like:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<apollo version="1.2.0">
  <owner name="Test User">
    <console idps="0000AAAAAA111111 1111DDDDD8888888" psid="0000AAAAAA111111 1111DDDDD8888888"/>
    <user id="00000001" account_id="ffffffffffffffff"/>
  </owner>
  <owner name="Some User">
    <console idps="0000AAAAAA111111 1111DDDDD8888888" psid="FFFFFFFFFF111111 000000DDDD888888"/>
    <user id="00000008" account_id="aaaaaaaaaaaaaaaa"/>
  </owner>
  <owner name="PS3 User">
    <console idps="0000AAAAAA111111 1111DDDDD8888888" psid="FFFFFFFFFF111111 000000DDDD888888"/>
    <user id="00000002" account_id="aaaaaaaaaaaaaaaa"/>
  </owner>
</apollo>
```

**Comment by graschbaten** [comment]
Thank You!

With the owners.xml everything worked flawless…

i will test few things soon why doesn’t work this without this file

I wish you have a cool breeze in this heat!
Thank you and have a nice week!

kind regards
Christian

**Comment by bucanero** [comment]
thanks for the feedback 👍 
good to know that using the `owners.xml` worked for you.

if you get more information about the issue with the auto-detect settings please let me know.
